src/compiler/emitter.ts

```diff
@@ -454,6 +454,14 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 [ModuleKind.CommonJS]: emitCommonJSModule,
             };
 
+            let bundleEmitDelegates: Map<(node: SourceFile, resolvePath?: boolean) => void> = {
+                [ModuleKind.ES6]() {},
+                [ModuleKind.AMD]: emitAMDModule,
+                [ModuleKind.System]: emitSystemModule,
+                [ModuleKind.UMD]() {},

```

Comment:
only the amd compat part of umd would work - trying to load the concatenated umd file in a commonjs  environment would only lead to sadness. So there's no point in allowing it.